Makefile

```diff
@@ -90,6 +90,9 @@ install-js-dev: node-version-check
 	yarn check --verify-tree || yarn install --check-files
 
 install-py-dev: ensure-pinned-pip
+	# The Python version installed via pyenv does not come with wheel pre-installed
+	# Installing wheel will speed up installation of Python dependencies
+	$(PIP) install wheel
```

Comment:
This guarantees that `wheel` is always installed before trying to install dependencies.

Reply:
Do we need to install this every time we call `install-py-dev`?

Reply:
I can check if it's already installed and if not then install it. Since we're not upgrading, we would hit the cache immediately.

Another approach would be to create a `make create-venv` step and make sure it happens at that point.

Reply:
Let me know your preference.